manager/gui/src/components/object-store-config/TenantSecurity.vue

```diff
@@ -0,0 +1,154 @@
+<!--
+* CORTX-Halo: CORTX-Halo Management GUI.
+* Copyright (c) 2022 Seagate Technology LLC and/or its Affiliates
+* This program is free software: you can redistribute it and/or modify
+* it under the terms of the GNU Affero General Public License as published
+* by the Free Software Foundation, either version 3 of the License, or
+* (at your option) any later version.
+* This program is distributed in the hope that it will be useful,
+* but WITHOUT ANY WARRANTY; without even the implied warranty of
+* MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+* GNU Affero General Public License for more details.
+* You should have received a copy of the GNU Affero General Public License
+* along with this program. If not, see <https://www.gnu.org/licenses/>.
+* For any questions about this software or licensing,
+* please email opensource@seagate.com.
+-->
+<template>
+  <div>
+    <div class="page-title">
+        Security
+    </div>
+    <v-divider></v-divider>
+    <div class="row-padding">
+      <v-row class="field-row">
+        <v-col cols="3" class="field-label">
+          Enable TLS <SgtTooltipIcon>
+                  Enable TLS for the tenant, this is required for Encryption Configuration
+                </SgtTooltipIcon> 
+        </v-col>
+        <v-col cols="2" class="toggle-col">
+           <SgtToggle
+            :tooltip=" toggleStates.enableTls ? 'On' : 'Off'"
+            v-model="toggleStates.enableTls"
+            :labelPre = "txtOff"
+            :labelPost = "txtOn"
+            :areToggleLabel="true"
+          ></SgtToggle>
+        </v-col>
+      </v-row>
+      <v-row class="field-row">
+        <v-col cols="3" class="field-label">
+          AutoCert <SgtTooltipIcon>
+                  MinIO Operator will generate all TLS certificates automatically.<br>Custom certificates: Allow user to provide your own certificates
+                </SgtTooltipIcon> 
+        </v-col>
+        <v-col cols="2">
+        </v-col>
+      </v-row>
+      <v-row class="field-row pl-4">
+        <v-col cols="3" class="field-label">
+          Enable AutoCert
+        </v-col>
+        <v-col cols="2" class="toggle-col">
+           <SgtToggle
+            :tooltip=" toggleStates.enableAutoCert ? 'On' : 'Off'"
+            v-model="toggleStates.enableAutoCert"
+            :labelPre = "txtOff"
+            :labelPost = "txtOn"
+            :areToggleLabel="true"
+          ></SgtToggle>
+        </v-col>
+      </v-row>
+      <v-row class="field-row pl-4">
+        <v-col cols="3" class="field-label">
```

Comment:
Remove pl-4 from this row and add pl-8 class in front of field-label class so the padding will look as per design

Reply:
Done.